libr/core/rvc.c

```diff
@@ -1183,16 +1238,31 @@ R_API int rvc_git_commit(RCore *core, const char *rp, const char *message, const
 	return r_vc_git_commit (rp, message);
 }
 
-R_API int rvc_git_branch(RCore *core, const char *rp, const char *bname) {
+R_API int rvc_git_branch(const RCore *core, const char *rp, const char *bname) {
 	if (!strcmp (r_config_get (core->config, "prj.vc.type"), "rvc")) {
 		return r_vc_branch (rp, bname);
 	}
 	return r_vc_git_branch (rp, bname);
 }
 
-R_API int rvc_git_checkout(RCore *core, const char *rp, const char *bname) {
+R_API int rvc_git_checkout(const RCore *core, const char *rp, const char *bname) {
 	if (!strcmp (r_config_get (core->config, "prj.vc.type"), "rvc")) {
 		return r_vc_checkout (rp, bname);
 	}
 	return r_vc_git_checkout (rp, bname);
 }
+
+R_API bool rvc_git_repo_exists(const RCore *core, const char *rp) {
+	char *frp;
+	if (!strcmp (r_config_get (core->config, "prj.vc.type"), "rvc")) {
+		frp = r_str_newf ("%s" R_SYS_DIR ".rvc", rp);
+	} else {
+		frp = r_str_newf ("%s" R_SYS_DIR ".rvc", rp);
+	}
+	if (!frp) {
+		return false;
```

Comment:
Be positive

Reply:
Well I am AB+...
no seriously what do you mean